Source/EasyNetQ.Tests/FluentConfiguration/PublishConfigurationTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using EasyNetQ.FluentConfiguration;
+using NUnit.Framework;
+
+namespace EasyNetQ.Tests.FluentConfiguration
+{
+    [TestFixture]
+    public class PublishConfigurationTests
+    {
+        [Test]
+        public void Should_throw_if_default_topic_is_null()
+        {
+            Assert.That(() => new PublishConfiguration(null), Throws.InstanceOf<ArgumentNullException>());
```

Comment:
I think yes, because if you don't want to use topic you should not pass
nothing, probably the same as passing null, but in fact WithTopic(null)
does not mean nothing is the default behavior.

Il lunedì 19 ottobre 2015, Kirill Teplinskiy <notifications@github.com> ha
scritto:

> In Source/EasyNetQ.Tests/FluentConfiguration/PublishConfigurationTests.cs
> <https://github.com/EasyNetQ/EasyNetQ/pull/495#discussion_r42360749>:
>
> > @@ -0,0 +1,36 @@
> > +﻿using System;
> > +using EasyNetQ.FluentConfiguration;
> > +using NUnit.Framework;
> > +
> > +namespace EasyNetQ.Tests.FluentConfiguration
> > +{
> > +    [TestFixture]
> > +    public class PublishConfigurationTests
> > +    {
> > +        [Test]
> > +        public void Should_throw_if_default_topic_is_null()
> > +        {
> > +            Assert.That(() => new PublishConfiguration(null), Throws.InstanceOf<ArgumentNullException>());
>
> Not sure is this test necessary or not.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EasyNetQ/EasyNetQ/pull/495/files#r42360749>.
>


-- 
Michael Denny
Engage IT Services s.r.l. <http://www.engageitservices.it/>
Software Architect, Trainer and Consultant
Microsoft MVP .NET <http://mvp.microsoft.com/mvp/Michael%20Denny-5000735>
More about me <http://about.me/micdenny>